src/test/java/org/reaktivity/nukleus/http_cache/internal/streams/proxy/ProxyExceptionsIT.java

```diff
@@ -83,7 +83,7 @@ public void shouldHandleAbortSentOnCacheableRequest() throws Exception
     @Test
     @Specification({
         "${route}/proxy/controller",
-        "${streams}/connect.reply.sent.abort/accept/client",
+         "${streams}/connect.reply.sent.abort/accept/client",
```

Comment:
Unintentional extra whitespace

Reply:
Done.

Reply:
The vertical misalignment still seems to be there, could you double check please?